test/platform-routes.test.js

```diff
@@ -64,9 +64,37 @@ describe('Platform Routes HTTP request handler', () => {
       await assertResponseStatus(res, 200)
       const metrics = await res.json()
       assert.deepStrictEqual(metrics, [
-        { day: '2024-01-11', station_id: 'station2' },
-        { day: '2024-01-12', station_id: 'station2' },
-        { day: '2024-01-12', station_id: 'station3' }
+        { day: '2024-01-11', station_id_count: 1 },
+        { day: '2024-01-12', station_id_count: 2 }
+      ])
+    })
+  })
+
+  describe('GET /stations/monthly', () => {
+    it('returns monthly station metrics for the given date range ignoring the day number', async () => {
+      // before the date range
+      await givenDailyStationMetrics(pgPool, '2023-12-31', ['station1'])
+      // in the date range
+      await givenDailyStationMetrics(pgPool, '2024-01-10', ['station1'])
+      await givenDailyStationMetrics(pgPool, '2024-01-11', ['station2'])
+      await givenDailyStationMetrics(pgPool, '2024-01-12', ['station2', 'station3'])
+      await givenDailyStationMetrics(pgPool, '2024-02-13', ['station1'])
+      // after the date range
+      await givenDailyStationMetrics(pgPool, '2024-03-01', ['station1'])
+
+      const res = await fetch(
+        new URL(
+          '/stations/monthly?from=2024-01-11&to=2024-02-11',
+          baseUrl
+        ), {
+          redirect: 'manual'
+        }
+      )
+      await assertResponseStatus(res, 200)
+      const metrics = await res.json()
+      assert.deepStrictEqual(metrics, [
+        { month: '2024-01-01', station_id_count: 3 },
```

Comment:
I have mixed feelings about the name `station_id_count`. I slightly prefer to use `stations` (we use `participants` in `GET /participants/daily`) or even `count`.

However, I also see how `station_id_count` is a very descriptive & accurate name, so I am fine to keep it.

Reply:
I like how precise it is, also it is a bit leaky (potentially exposes too many internals). I suggest we iterate on this at a later point, when we understand our system better